brew.py

```diff
@@ -1,104 +1,126 @@
-import os, platform, subprocess, dotbot
-
+import os
+import platform
+import subprocess
+import dotbot
 class Brew(dotbot.Plugin):
     _brewDirective = "brew"
     _caskDirective = "cask"
     _tapDirective = "tap"
     _brewFileDirective = "brewfile"
+    _defaults = {}
```

Comment:
I don't like these two variables. There is no reason for them to be mutable. Would be nicer to get defaults and pass them to directives as an argument (e.g. `self._brew(data, defaults)`). This way it's easier to read the code and there are no unintended side effects.